packages/@aws-cdk/aws-apigateway/test/api-key.test.ts

```diff
@@ -282,11 +281,11 @@ describe('api key', () => {
 
       // THEN
       // should have an api key with no props defined.
-      expect(stack).toHaveResource('AWS::ApiGateway::ApiKey', undefined, ResourcePart.CompleteDefinition);
+      Template.fromStack(stack).hasResource('AWS::ApiGateway::ApiKey', Match.anyValue());
```

Comment:
I feel like this isn't exactly "should have an api key with no props defined". We should be able to assert this (reference [PR](https://github.com/aws/aws-cdk/pull/16678)):

```ts
Template.fromStack(stack).hasResourceProperties('AWS::ApiGateway::ApiKey', Match.absent());
```

Reply:
Ah, makes sense. I'll fix it.